hibernate-core/src/main/java/org/hibernate/internal/StatelessSessionImpl.java

```diff
@@ -298,28 +299,61 @@ public Object internalLoad(
 			// caller did not request forceful eager loading, see if we can create
 			// some form of proxy
 
+			final boolean allowBytecodeProxy = getFactory()
+					.getSessionFactoryOptions()
+					.isEnhancementAsProxyEnabled();
+
```

Comment:
`StatelessSessionImpl` has a property `allowBytecodeProxy`, no need to recalculate it.

Reply:
oh, ok; thanks, I'll get rid of that